strings/charoccurrence.go

```diff
@@ -0,0 +1,18 @@
+// charoccurrence.go
+// description: Chars occurrence in the sentence
+// details: An algorithm to count chars in string
+// author(s) [Moein](https://github.com/mo1ein)
+// see charoccurrence_test.go
+
+package strings
+
+func Count(text string) map[rune]int {
+	charOccurrence := make(map[rune]int, 0)
```

Comment:
```suggestion
	charMap := make(map[rune]int, 0)
```

the name is too big. In Go, people usually prefer smaller names unless absolutely necessary to add more context. Here, its quite self explanatory.

Reply:
- [x] fixed